src/storage/keyvalue/JsonResourceStorage.ts

```diff
@@ -65,7 +66,17 @@ export class JsonResourceStorage implements KeyValueStorage<string, unknown> {
   }
 
   public async* entries(): AsyncIterableIterator<[string, unknown]> {
-    const container = await this.source.getRepresentation({ path: this.container }, {});
+    let container: Representation;
+    try {
+      container = await this.source.getRepresentation({ path: this.container }, {});
+    } catch (error: unknown) {
+      // Container might not exist yet
```

Comment:
So we don't create? Perhaps extend the comment to explain why non-existence is not an issue.

Reply:
I'll add a comment. The reason is that we use `setRepresentation` when writing data, which will create the container if it doesn't exist yet, so there's no need to have it if there is no data.